### PR TITLE
Blr/chores/clean readme and minor lint

### PR DESCRIPTION
**Pull request by bartriordan**
@BJK @maneeshanand Please review. This PR:
- Cleans up the `README` by adding an anchor-linked list of sections, cleaning up grammar, cleaning up code layout, and adding JavaScript formatting for code blocks; and,
- Does some minor linting (e.g., use object method shorthand).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,68 @@
-Fluxxed_up
+Fluxxed_up [![Build Status](https://travis-ci.org/everplans/fluxxed_up.svg?branch=master)](https://travis-ci.org/everplans/fluxxed_up) [![Code Climate](https://codeclimate.com/repos/57361f9378bbd61943002b6b/badges/59cae8b81ec09ab798fb/gpa.svg)](https://codeclimate.com/repos/57361f9378bbd61943002b6b/feed) [![Dependency Status](https://gemnasium.com/badges/github.com/everplans/fluxxed_up.svg)](https://gemnasium.com/github.com/everplans/fluxxed_up)
```
Comment (bartriordan) [inline README.md:1]: Moved these inline with the title to follow Facebook's style for the [React `README`](https://github.com/facebook/react/blob/master/README.md)

File: README.md [hunk 2]
```diff
@@ -1,60 +1,73 @@
...
-* If you are using fluxxed_up from source, you'll need to have babel setup in your project, since the non-built code only exposes an es6 entry point. If you'd don't have babel, you'll need to run `npm run build` everytime you want to see your changes.
+## Build Process
+If you are using fluxxed_up from source, you'll need to have Babel setup in your project since the unbuilt code only exposes an ES6 entry point. If you don't have Babel, you'll need to run `npm run build` every time you want to see your changes.
 
-## What's included:
+## What's Included
 
-* Action Prototype
-Based of off facebook patterns and intended to be used with flux. It works with a loose $.ajax wrapper to fire
+##### ActionPrototype
```
Comment (bartriordan) [inline README.md]: Changed these to be `h5`s so I could link to the auto-generated anchor tags from the table of contents

File: README.md [hunk 3]
```diff
@@ -1,60 +1,73 @@
...
-  Types: KeyMirror({
-    GOT_USERS: null
-  }),
-  fetchUsers: function() {
-     this.fireApi('get', 'assessment', null,
-      {successAction: this.Types.GOT_ASK_DOCUMENT_GROUPS, //Note that since no failureAction was specified, it will dispatch this action always.
-      JSONHead: 'assessment'}) //head of the json coming back from the server.
+own actions. You specific a REST actions (`GET`, `POST`, etc), and endpoint, and optional data payload (for a `PATCH`, `POST`, or `PUT`). You can optionally supply action types to dispatch based on a success or failure from the server. Here's an example of how to construct an action with `ActionPrototype`:
+
+```javascript
```
Comment (bartriordan) [inline README.md:36]: This adds syntax highlighting in GitHub-flavored markdown

**Comment by maneeshanand** [comment]
this looks GTM 👍

**Comment by bartriordan** [comment]
@maneeshanand Thanks! Merging.

**Pull request merged**